solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -238,7 +238,7 @@ client.add(Arrays.asList(d1, d2));
 
 == Query Time
 
-Apache Solr provides two query parsers that work with dense vector fields, that each support differnet ways of matching documents based on vector similarity: The `knn` query parser, and the `vecSim` query parser.
+Apache Solr provides two query parsers that work with dense vector fields, that each support different ways of matching documents based on vector similarity: The `knn` query parser, and the `vecSim` query parser.
```

Comment:
vecSim -> vectorSimilarity

Reply:
Oh man ... i'm not sure how I managed to forget to do all the renames in the docs, i'll fix.